doc/conf.py

```diff
@@ -130,9 +130,19 @@
     'within_subsection_order': FileNameSortKey,
     'ignore_pattern': '/generate_',
     'nested_sections': False,
-    'copyfile_regex': r'.*\.rst|.*\.png|.*\.svg'
+    'copyfile_regex': r'.*\.rst|.*\.png|.*\.svg',
```

Comment:
Hello, sorry I missed this.

This copies any rst, png or svg files from the source `sphinx-gallery` directory to the output directory. In this case the source directory is `examples/tutorials/` and we would like it to copy `waveform_extractor_to_sorting_analyzer.rst` from there to where the gallery is outputted (`doc/tutorials`). 

This is done so that the tutorial pages (https://spikeinterface.readthedocs.io/en/latest/tutorials/index.html), which is a sphinx-gallery, can include a non-sphinx gallery page (the waveforms page).

The `.svg`s were included to copy over the svg of the extension parent/child figure, but this is not kept in `doc/_images` since it's repeated in a few places.